docs/ui/preferences/container-engine.md

```diff
@@ -6,12 +6,45 @@ title: Container engine
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## General
+
 Set the [container runtime] for Rancher Desktop. Users have the option of [containerd] which provides namespaces for containers and the use of nerdctl or [dockerd (moby)] which enables the Docker API and the use of the Docker CLI. Only one container runtime will function at a time.
 
 When switching to a different container runtime:
 
 - Workloads and images that have been built or pulled using the existing container runtime are not available on the container runtime being switched to.
 
+## Allowed Images
+
+The `Allowed Images` tab provides options to let you control which registry artifacts you can access within Rancher Desktop. For example, you may want to pull container images only from your organization's private registry or only from your department-specific namespace in your organization's registry, etc. You can specify image URL patterns to allow accessing images only from specific registries and/or repositories.
+
+Check the **Enable** checkbox to enable Rancher Desktop to apply the specified patterns while pulling images, which means only the image URLs that match atleast one of the specified patterns will be allowed.
+
+You can use the `+` and `-` buttons to add/remove image URL patterns. 
+
+![](/img/preferences/allowed-images.png)
+
+### How to specify Allowed Image patterns
+
+You can specify Allowed Image patterns using the format `[registryServerUrl/][:port/][organization/]repository[:tag]`. Rancher Desktop supports using regular expressions in any part of an Allowed Image pattern string.
+
+> If not specified in an Allowed Image pattern string,
+> * `registryServerUrl` defaults to DockerHub registry (`registry-1.docker.io`)
+> * `port` defaults to 433, but is optional because the HTTP "Host" header may not include it. 
+> * `organization` for DockerHub registry defaults to `library`, and for Non-DockerHub registries defaults to no organization.
+> * `tag` defaults to anything (not including a '/') and NOT just to "latest".
```

Comment:
Is the `/` supposed to be formatted/enclosed with backticks instead of single quotes? In other instances, double quotes are used for quoting.